host-configs/hpcmp/scout/gcc-8.3.1-cuda-11-openmpi.cmake

```diff
@@ -0,0 +1,23 @@
+#------------------------------------------------------------------------------
+# Compiler Settings
+#------------------------------------------------------------------------------
+set(CMAKE_C_COMPILER /usr/bin/gcc CACHE PATH "C compiler to use" )
+set(CMAKE_CXX_COMPILER /usr/bin/g++ CACHE PATH "C++ compiler to use")
+set(CMAKE_Fortran_COMPILER /usr/bin/gfortran CACHE PATH "Fortran compiler to use")
+
+#------------------------------------------------------------------------------
+# CUDA 
+#------------------------------------------------------------------------------
+set(CMAKE_CUDA_ARCHITECTURES "70" CACHE STRING "")
+#set(CUDA_ARCHITECTURES "70" CACHE)
+
+#------------------------------------------------------------------------------
+# MPI
+#------------------------------------------------------------------------------
+set(MPI_BIN_ROOT /p/app/restricted/create/av/helios/externals/scout_tools/openmpi-4.0.5/bin CACHE PATH "MPI bin PATH")
```

Comment:
@jsitaraman -- It looks like this version of Open MPI is built with CUDA-Aware support:
```
16:53 $ ompi_info --parsable --all | grep mpi_built_with_cuda_support:value
mca:mpi:base:param:mpi_built_with_cuda_support:value:true
```